downstream/assemblies/platform/assembly-network-ports-protocols.adoc

```diff
@@ -34,14 +34,28 @@ The following default destination ports and installer inventory listed are confi
 | 22 | TCP | SSH | Installer node | Hybrid node | Installation (temporary) | `ansible_port`
 | 22 | TCP | SSH | Installer node | PostgreSQL database| Remote access during installation (temporary) | `pg_port`
 | 80/443 | TCP | HTTP/HTTPS | Installer node | {HubNameStart} | Allows installer node to push the execution environment image to {HubName} when using the bundle installer. | Fixed value
+| 80/443 | TCP | HTTP/HTTPS | EDA node | {HubNameStart} | | Fixed value
+| 80/443 | TCP | HTTP/HTTPS | EDA node | {ControllerNameStart} |  | Fixed value
+| 80/443 | TCP | HTTP/HTTPS | {ControllerNameStart} | {HubNameStart} |  | Fixed value
 | 80/443 | TCP | HTTP/HTTPS | Execution node | {HubNameStart} | Allows execution nodes to pull the execution environment image from {HubName}. | Fixed value
 | 443 | TCP | HTTPS | Controller node | Client | Web UI/API
 
 This exposes the mesh ingress receptor entry point for inbound connections.| `nginx_https_port`
 | 443 | TCP | HTTPS | Controller node | {OCPShort} |  Only required when using container groups to run jobs. | Host name of OpenShift API server
+| 443 | TCP | HTTPS | HA Proxy load balancer | Platform gateway |  This is the ingress above the gateway that's customer controlled and can load balance requests to multiple gateways. | 
+| 443 | TCP | HTTPS | Platform gateway | {ControllerNameStart} |  | 
+| 443 | TCP | HTTPS | Platform gateway | {HubNameStart} |  | 
+| 443 | TCP | HTTPS | Platform gateway | {EDAName} |  | 
+| 443 | HTTPS | Receptor | Execution node | OCP Mesh ingress |  | 
+| 443 | HTTPS | Receptor | Hop node | OCP Mesh ingress |  | 
 | 5432 | TCP | PostgreSQL | Controller node | PostgreSQL database | Open only if the internal database is used along with another component. Otherwise, this port should not be open. | `automationcontroller_pg_port`
 | 5432 | TCP | PostgreSQL | EDA node | PostgreSQL database | Open only if the internal database is used along with another component. Otherwise, this port should not be open. | `automationedacontroller_pg_port`
 | 5432 | TCP | PostgreSQL | {HubNameStart} | PostgreSQL database | Open only if the internal database is used along with another component. Otherwise, this port should not be open. | `automationhub_pg_port`
+| 5432 | TCP | PostgreSQL | Platform gateway | External database |  | `automationgateway_pg_port`
```

Comment:
Possibly the description for this should be the same as for the others, ' Open only if the internal database is used along with another component. Otherwise, this port should not be open.'

Reply:
Thanks, Ian!

@matburt could you weigh in here?

Reply:
Yeah that's fair

Reply:
Thank you both! I have made that change.